packages/components/psammead-image/src/index.jsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Image = styled.img`
+  display: block;
+  width: 100%;
+`;
+
+Image.propTypes = {
+  alt: PropTypes.string.isRequired,
+  height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
```

Comment:
I think our linting isn't working as expected on this repo. Destructuring would make propTypes more readable.
Also, we should either enforce strings or numbers to be passed in for `height` `width` props. I don't mind which it is, but having consistency would be good. 

```
import { number, string } from 'prop-types'; 
...
Image.propTypes = {
  alt: string.isRequired,
  height: number.isRequired,
  src: string.isRequired,
  width: number.isRequired,
};
```

Reply:
+1 for numbers 👍